qa/tasks/ceph.py

```diff
@@ -1632,9 +1632,13 @@ def restart(ctx, config):
         for dmon in daemons:
```

Comment:
i would log the daemons list.. not sure why the existing code wouldn't work.

and/or, try just listing the osds explicitly and see if that works around the problem.